docs/9-platform-engineering/9.1.1-software-catalog.md

```diff
@@ -0,0 +1,84 @@
+      estMinutes: 120
+      technologies:
+        - Backstage
+        - GitHub
+
+    - name: OpenAPI Definitions
+      description: |
+        Here, students are required to provide an OpenAPI specification which their Backstage instance can ingest and associate with a definition for an API having source code in a repo owned by the student's organization.  Students may use a repo that already has an Open API spec file or they may generate the necessary spec data from an API.  
+      estMinutes: 30
+      technologies:
+        - Backstage
+        - GitHub
+---
+
+# Backstage Software Catalog
+
+The Software Catalog is a key feature of Backstage.  Please read more about it on the [Backstage website](https://backstage.io/docs/features/software-catalog/).
+
+### Exercise 1: Register an entity to Backstage
+
+In Backstage, entities represent different types of resources within your system.  They are defined using YAML files and registered in the Backstage catalog.  Each entity type has a specific purpose and schema.  
+
+1. Following Backstage's [Getting Started](https://backstage.spotify.com/learn/standing-up-backstage/putting-backstage-into-action/8-integration/) guide, add a GitHub integration to your Backstage instance.
+
+  ?> These instructions use a personal access token to authenticate Backstage with GitHub, but you may also use a [GitHub App](https://backstage.io/docs/integrations/github/github-apps).  
+
+2. Continue following the above guide to manually register a component entity to your Backstage instance.  *You may find it helpful for future exercises to take note of the information that is provided alongside your components in the UI, and consider where it comes from.*
+
+### Exercise 2: Configure automatic catalog ingestion
+
+1. Now that you have manually registered an entity to Backstage, configure your Backstage instance to automatically register entities based on repos that it detects within a GitHub organization that you own.  
+
+  ?> See [GitHub Discovery](https://backstage.io/docs/integrations/github/discovery/) for a useful guide.
+
+### Exercise 3: Generate a dependency graph
+
+One useful feature of Backstage is its ability to leverage `catalog-info.yaml` files to automatically generate dependency graphs depicting the relationship between entities in a software ecosystem.  The visual aid provided by these graphs can help users quickly understand complex systems.
+
+1. Fork two interdependent repos, such as [dks-ui](https://github.com/liatrio/dks-ui) and [dks-api](https://github.com/liatrio/dks-api).
+
+  ?> Using `dks-api` or another repo that provides easy access to Open API specs will make exercise 4 much easier.
```

Comment:
I feel like we should just point them to dks-ui and dks-api vs suggesting it.

Reply:
Done